### ISSUE TITLE
pass-occurrences_override_the_type_classification_per_system_1_3

### ISSUE THREAD

**Issue by rubendel** [buildingSMART/IDS#108]
I think for this test case there is one ```IFCWALL``` too many. I think #4 should be removed because it makes the testcase fail.

Probably the same thing for pass-occurrences_override_the_type_classification_per_system_3_3.

Created an issue as PR will eventually get overwritten.

**Comment by hesrah** [comment]
There is a `/* Testcase */` marker for the line relevant to the example.

But in general, why are the examples for Classification so long and full of "junk"? Could we shorten this example like for the attribute test cases etc. (without `/* Testcase */` mark unless absolutely necessary). It would make it much more readable? It's often quite confusing. The other test cases (Attribute etc.) are very smart.

**Comment by rubendel** [comment]
That marker is not in the actual test case IFC file (the ones in https://github.com/buildingSMART/IDS/tree/master/Documentation/testcases).

I hope the marker is only there for us human beings, not for the computer to recognize.

A few types could be omitted from this example I guess, but you definitely want to keep the occurrence and the type there.

**Comment by aothms** [comment]
I agree, the trick is that from fail/pass in the file name you should be able to detect the expected outcome. A comment marker to highlight a specific focus entity is indeed not very portable.

@Moult any ideas on a quick fix? Are there any cases where multiple products in the file is a necessity?

**Comment by rubendel** [comment]
Just to clarify, in my experience for >90% of the testcases the fail/pass indicator in the filename does seem to cover the whole IDS, so any extra entities are not interfering with that. Got to say I really appreciate all the effort that must have gone into this @Moult (and @aothms I presume?).

**Comment by pasi-paasiala** [comment]
Fix the IFC file for 1.0.

**Comment by CBenghi** [comment]
I think I fixed the file a couple of commits back without being aware of this issue.
Could somebody review the current state of the file in the PR #240 and close this if appropriate?

**Comment by andyward** [comment]
Confirmed this is **partially** fixed with PR #240 

There's now only one IfcWall in the 'pass' test files
_pass-occurrences_override_the_type_classification_per_system_1_3_ and _pass-occurrences_override_the_type_classification_per_system_3_3_

However the extra wall still exists in _fail-occurrences_override_the_type_classification_per_system_2_3_ and I think it's possible this may invalidate the test and create false negatives. E.g. The test is expected to fail because the requirement is '_All elements of entity Wall should have a classification value 22_' - and in this case the wall instance over-rides the value to '11' - but actually the test will always fail due to the extraneous wall does not meet the requirement.

Did you spotfix the IFC files or fix them at [source](https://github.com/IfcOpenShell/IfcOpenShell/blob/59fbf2bbde6d94c1d122b6dd7bc232e0440a6cc9/src/ifctester/test/test_facet.py#L823) @CBenghi ?

More generally I think these test IFCs for the Classification testcases need a slim down. There's a lot of redundant data in them from earlier tests which I think obscure the intent. e.g. just compare a property test with a classification test:

![image](https://github.com/buildingSMART/IDS/assets/923699/036bf373-1e53-413b-b732-63011a78675d)

**Comment by giuseppeverduciALMA** [comment]
I'll check this case
